app/scss/statics/_the-flex-grid.scss

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
Ah, this is the file I commented about. Don't include it, just import it from the node_modules directory.